flavour/simple/Dockerfile.flavour

```diff
@@ -1,6 +1,6 @@
 FROM alpine:latest
 
-RUN apk add sudo bash curl libc6-compat && \
+RUN apk add sudo bash curl libc6-compat ca-certificates && \
```

Comment:
This doesn't fit to the PR title

Reply:
Right. Those were changes that came during the tests. I'll fix the title